extensions/notebook/package.nls.json

```diff
@@ -3,6 +3,7 @@
 	"description": "Defines the Data-procotol based Notebook contribution and many Notebook commands and contributions.",
 	"notebook.configuration.title": "Notebook configuration",
 	"notebook.enabled.description": "Enable viewing notebook files using built-in notebook editor.",
+	"notebook.pythonPath.description": "Contains python installation path on user's machine.",
```

Comment:
"Contains" sounds a little weird here. Maybe just replace it with "The"? Also, how about "local machine" instead of "user's machine"?

Reply:
local machine sounds good. I was thinking like a verb "Saves/stores python installation path". Any idea?